Makefile

```
@@ -0,0 +1,13 @@
+REPO=amazon-mws-analytics
+
+.PHONY: build
+build:
+	docker build -t $(REPO):latest .
+
+.PHONY: style
+style:
+	docker run -it --rm -v "$(PWD)":/code jbbarth/black *.py
```

This image looks pretty out-of-date; it's two years old, and it looks like `black` releases more often than that: https://pypi.org/project/black/#history

Also, I'm obliged to yell about depending on random people's packages or Docker images. I don't like trusting it.

Maybe we could do the same thing we do below for the `lint` case (I don't know if this entrypoint will work for sure):
```suggestion
	docker build --target tools -t $(REPO):lint .
	docker run --rm -it --entrypoint="black" $(REPO):lint 
```

---

Yeah. I was just being lazy. I agree with all of your points, and I'll update it.